components/aiplatform-components/Pipfile

```diff
@@ -4,12 +4,12 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-kfp = "==1.8.9"
-google-cloud-pipeline-components = ">=1.0.0,<2.0.0"
-google-cloud-aiplatform = ">=1.24.1,<2.0.0"
+kfp = "==1.8.21"
 
 [dev-packages]
-pytest = ">=6.2.4,<7.0.0"
+google-cloud-aiplatform = ">= 1.24.1,<2.0.0"
+google-cloud-pipeline-components = ">= 1.0.0,<2.0.0"
```

Comment:
google-cloud-aiplatform and google-cloud-pipeline-components should be pinned to an exact version (to match `packages_to_install`)